mtc.go

```diff
@@ -488,12 +427,13 @@ func (p *CAParams) MarshalBinary() ([]byte, error) {
 	// TODO add struct to I-D
 	var b cryptobyte.Builder
 	b.AddUint8LengthPrefixed(func(b *cryptobyte.Builder) {
```

Comment:
Eg here

Reply:
To add even more to this confusion: Here it should be correct again. Please note the difference: In this case, I'm adding the raw bytes of the `RelativeOID` - just a type alias for `[]uint8` which does not (yet) implement a marshal method - while in the case discussed previously, I use the `MarshalBinary()` method of `TrustAnchorIdentifier`. Would you like me to align these usages, i.e., implement `MarshalBinary()` for `RelativeOID`?

Reply:
Ah, ofc. Aligning them would be better.

Reply:
See #6